### ISSUE TITLE
Sequential execution of multiple requests.

### ISSUE THREAD

**Issue by chaeron** [Huachao/vscode-restclient#724]
Since it's possible to name requests, it would be really useful if there was some way to specify sequences of requests that you want to run one after another.

Say we have a .rest file like this:

```
// @name request1

GET http://{{server}}:{{port}}/{{api_prefix}}/first_request HTTP/1.1
Content-Type: {{content_type}}

###

@returned_value = {{request1.response.body.returned_value}}

// @name request2

GET http://{{server}}:{{port}}/{{api_prefix}}/sceond_request/{{returned_value}} HTTP/1.1
Content-Type: {{content_type}}

###
```

The it would be great if we could specify a sequence of requests somehow, maybe like this:

`// @sequence request1 request2`

And have restclient show a clickable "Send Requests" above the sequence that when clicked would execute request1 followed immediately by request 2.

Each request could still open a separate tab to show it's response value.

**Comment by chadbr** [comment]
duplicate of #199 ?

**Comment by chaeron** [comment]
Not really a duplicate.....since I would like to see all requests in one file, but then a mechanism to specify multiple/different sequences that you want to run that depend on those requests.

It's very possible a request would be included in multiple sequences.

This is beyond 199

**Comment by chaeron** [comment]
Still waiting for a response on this.  Much as we like REST Client, the inability to run a sequence of requests limits it's usefulness.

**Comment by flyneth** [comment]
duplicate of #444

**Comment by chaeron** [comment]
> duplicate of #444

This is NOT a duplicate of 444, as I noted in a prior comment.  444 is to execute all requests in a file. I do not want/need that. I want to be able to specify multiple/different sequences that I want to run requests that are specified in the file.

**Comment by liflovs** [comment]
Agree, was investigating its applicability and found it useful only for quick checks without chaining

**Comment by ryan-efendy** [comment]
what's the latest on this?

cc: @chaeron

**Comment by HiveTechDev** [comment]
This is one of the "last" big things keeping me from changing all our internal rest things to this. would be so nice to run jwt login and then the get in one button click

**Comment by henriec** [comment]
This would be the best feature to add!
+1 from me.

**Comment by blackfyre** [comment]
I'm guessing that this went nowhere since posting?

**Comment by chaeron** [comment]
> I'm guessing that this went nowhere since posting?

Nada. Not impressed with lack of response from developer.
